docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md

```diff
@@ -87,6 +87,305 @@ For more information about ACP payment methods integration with your project OMS
 
 ## Checkout payment step
 
+Implementing the Payment Provider Stripe into your project can be done in many different ways.
+
+- You can add it in your own headless frontend application using Glue.
+
+[//]: # (- You can use the default implementation in the Payment selection page using Yves which shows the Payment Elements then later on the summary page.)
+- You can add it as a hosted Payment page which uses a redirect after your customer submits the order.
+
+### Headless implementation
+
+This approach should be used when your project is a headless project without an Yves application. Make sure you have the following modules installed in the specified or newer versions:
+
+#### Modules
+
+- `spryker/kernel-app:1.2.0`
+- `spryker/payment:5.24.0`
+- `spryker/payments-rest-api:1.3.0`
+
+#### PreOrder payment flow in a Nutshell
+
+- The customer either selects Stripe as the payment method or he gets only Stripe Elements presented.
+- When you have more than one Payment Provider in your project and Stripe gets selected the `InitializePreOrderPayment` Glue API endpoint (glue.your-website.com/payments?action=initialize-pre-order-payment) is called with the Payment Provider name (Stripe), the Payment Method name (Stripe), CSS-ID of the element to render Stripe Elements to, CSS-ID of the element to render messages to, the Payment Element Options including the `return_url`, and the Quote data. See example below.
+- When you only use Stripe in your project then the `InitializePreOrderPayment` Glue API endpoint (glue.your-website.com/payments) is called with the Payment Provider name (Stripe) and the Payment Method name (Stripe). See example below.
```

Comment:
quote and payment amount is still needed, so there is no difference with the previous paragraph, should be combined into one